framework/sessions/__init__.py

```diff
@@ -77,6 +76,7 @@ def prepare_private_key():
 
 
 def get_session():
+    Session = apps.get_model('osf.Session')
```

Comment:
IIRC, importing it at the top of this file broke app initialization. I'll test that to confirm.